awxkit/awxkit/cli/custom.py

```diff
@@ -56,6 +56,7 @@ def add_arguments(self, parser, resource_options_parser, with_pk=True):
         parser.choices[self.action].add_argument('--monitor', action='store_true', help='If set, prints stdout of the launched job until it finishes.')
         parser.choices[self.action].add_argument('--action-timeout', type=int, help='If set with --monitor or --wait, time out waiting on job completion.')
         parser.choices[self.action].add_argument('--wait', action='store_true', help='If set, waits until the launched job finishes.')
+        parser.choices[self.action].add_argument('--interval', type=int, help='If set with --monitor or --wait, amount of time to wait between api calls.')
```

Comment:
The unit is not mentioned here. Assuming it is in seconds but the default value below in `float`.